tests/test_FastApiPacker.py

```diff
@@ -0,0 +1,258 @@
+import os
+import pytest
+import shutil
+import json
+import yaml
+from unittest.mock import patch, mock_open, MagicMock
+from src.ersilia_pack.packer import FastApiAppPacker
+
+@pytest.fixture
+def temp_model_directory(tmp_path):
+    """Fixture to create a temporary model directory."""
+    model_dir = tmp_path / "test_model"
+    os.makedirs(model_dir / "model")
+    os.makedirs(model_dir / "framework")
+    return model_dir
+
+@pytest.fixture
+def bundles_repo_path(tmp_path):
+    """Fixture to create a bundles repository path."""
+    return tmp_path / "bundles_repo"
+
+# 1. Test correct initialization
+def test_fastapi_packer_initialization(temp_model_directory, bundles_repo_path):
+    metadata_file = temp_model_directory / "metadata.json"
+    metadata_file.write_text(json.dumps({"Identifier": "test_model_id"}))
```

Comment:
Can we mimic the side-effects to resemble more real world scenario? Like `eos3b5e` or something as the identifier.